backends/dpdk/dpdkProgram.cpp

```diff
@@ -187,6 +187,79 @@ IR::Declaration_Variable *ConvertToDpdkParser::addNewTmpVarToMetadata(cstring na
     return newTmpVar;
 }
 
+/* This is a helper function for handling the transition select statement. It populates the left
+   and right operands of comparison, to be used  in conditional jump instructions. When the keyset
+   of select case is simple expressions, it populates the left and rigt operands with the input
+   expressions. When the keyset is Mask expression "a &&& b", it inserts temporary variables in
+   Metadata structure and populates the left and right operands of comparison with
+   "input & b" and "a & b" */
+void ConvertToDpdkParser::getCondVars(const IR::Expression *sv, const IR::Expression *ce,
+                                      IR::Expression **leftExpr, IR::Expression **rightExpr) {
+    if (sv->is<IR::Constant>() && sv->type->width_bits() > 32) {
+        ::error(ErrorType::ERR_UNEXPECTED,
```

Comment:
This is a back end constraint so perhaps you should use one of the unsupported errors.

Reply:
Updated